### PR TITLE
Added French to inYourLanguage.ts

### PR DESCRIPTION
**Pull request by Ascor8522**
# Changes

- Added entry for French in the `inYourLanguage.ts` file.
- Sorted the keys in the object (in the same file) by languages code.

### CODE REVIEW

**Review by Ascor8522 — COMMENTED** [review]
Why did the translation get changed? I translated it in a way very close to what's often used on internationalized websites.
Also, the closing curly bracket is now missing and it's making the tests fail.

**Comment by orta** [comment]
Bad rebase via the website - that one got merged before I saw yours come in and was going to ask your opinion. Happy to prefer yours 👍🏻

**Review by kleinkkk — APPROVED** [review]

**Review by kleinkkk — APPROVED** [review]

**Comment by kleinkkk** [comment]
> # Changes
> 
> - Added entry for French in the `inYourLanguage.ts` file.
> - Sorted the keys in the object (in the same file) by languages code.

3ea584228ebdbd262d76b580306b82ebc12b543e

**Comment by kleinkkk** [comment]
Jigjg

**Comment by kleinkkk** [comment]
c6ce5932484b52dc483f872925aec500df0636e0

**Comment by kleinkkk** [comment]
> # Changes
> 
> - Added entry for French in the `inYourLanguage.ts` file.
> - Sorted the keys in the object (in the same file) by languages code.

c6ce5932484b52dc483f872925aec500df0636e0

File: packages/typescriptlang-org/src/copy/inYourLanguage.ts [hunk 1]
```diff
@@ -20,34 +20,34 @@ export const inYourLanguage: Record<
     open: "Adelante",
     cancel: "No preguntar de nuevo",
   },
-  pt: {
-    shorthand: "Em Pt",
-    body: "Esta página está disponível em Português",
```
Comment (kleinkkk) [inline packages/typescriptlang-org/src/copy/inYourLanguage.ts:25]: c6ce5932484b52dc483f872925aec500df0636e0

**Review by kleinkkk — APPROVED** [review]

**Pull request merged**